apps/namadillo/src/App/Common/FiatCurrency.tsx

```diff
@@ -1,7 +1,15 @@
 import { Currency, CurrencyProps } from "@namada/components";
+import BigNumber from "bignumber.js";
 
 type FiatCurrencyProps = Omit<CurrencyProps, "currency">;
 
 export const FiatCurrency = (props: FiatCurrencyProps): JSX.Element => {
-  return <Currency currency={{ symbol: "$", fraction: "cents" }} {...props} />;
+  const formattedAmount = new BigNumber(props.amount.toPrecision(2));
```

Comment:
This is also rounding bigger numbers e.g. `BigNumber(123)` displays as `$120`. Maybe the `decimalPlaces` function will work instead?